### PR TITLE
This addresses #77 grunt-if dependency update

### PR DESCRIPTION
**Pull request by binarymist**
Tested with docker also

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ docker-compose up
 
 
 #### Customizing the Default Application Configuration
-The default application settings (database url, http port, etc.) can be changed by updating the [config file] (https://github.com/OWASP/NodejsGoat/blob/master/config/env/all.js).
+The default application settings (database url, http port, etc.) can be changed by updating the [config file] (https://github.com/OWASP/NodeGoat/blob/master/config/env/all.js).
```
Comment (ckarande) [inline README.md]: Apparently, an unintentional change.

File: package.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
     "grunt-jsbeautifier": "^0.2.12",
     "grunt-mocha-test": "^0.12.7",
     "grunt-nodemon": "^0.4.2",
-    "grunt-if": "^0.1.5",
+    "grunt-if": "https://github.com/binarymist/grunt-if/tarball/master",
```
Comment (ckarande) [inline package.json]: Have you modified grunt-if module? Can't we take it from npm?What is changed?

File: README.md (hunk 1 above)
Comment (binarymist) [inline README.md]: Well, I had changed it for my fork to binarymist, but then realised that NodejsGoat was no longer correct.

File: package.json (hunk 2 above)
Comment (binarymist) [inline package.json]: This addresses https://github.com/OWASP/NodeGoat/issues/77
The changes to grunt-if are here: https://github.com/binarymist/grunt-if/commit/01131e5c43c0c4198aa785192416098b1ba52a5e

It's basically dependencies. The maintainer is unresponsive, so the package on npm is no longer maintained. I simply forked and did what the package maintainer should have.

Comment (ckarande) [inline package.json]: ok.. thanks for reminding me about related issue.

**Pull request merged**